### PR TITLE
Updating with 2017 data

### PR DESCRIPTION
**Pull request by supercasey**

### CODE REVIEW

File: tests/testthat/test_may_load.R [hunk 1]
```diff
@@ -24,6 +25,7 @@ test_that("may load has the correct columns", {
 		"SITE_QW_ID",
 		"CONSTIT",
 		"WY",
+		"MONTH",
```
Comment (cschroed-usgs) [inline tests/testthat/test_may_load.R]: Adding columns to the data frames can require in changes to the database, server-side and client-side code. Is it necessary to add the MONTH column?

File: tests/testthat/test_annual_load.R [hunk 2]
```diff
@@ -109,8 +116,10 @@ test_that("There are no duplicate values", {
 
 test_that("Most recent water year has all of the necessary sites ", {
   temp_aloads_recent<-temp_aloads[temp_aloads$WY%in%max(temp_aloads$WY),] 
-  expected <- sort(c("HAZL","PADU","GRAN","CLIN","WAPE","KEOS","VALL","GRAF","SIDN","OMAH","ELKH","LOUI","DESO","HERM","THEB","SEDG","HARR","LITT","KERS",
-                     "STFR","BATO","BELL","MELV","CALU","MORG","SUMN","STTH","GULF","NEWH","CANN","MISS"))
+  expected <- sort(c("HAZL","PADU","GRAN","CLIN","WAPE","KEOS","VALL","GRAF","SIDN","OMAH","ELKH","LOUI","DESO","HERM","THEB","SEDG","HARR","KERS","MORG","BELL",
+                     "STFR","MELV","SUMN","STTH","GULF","NEWH","CANN","MISS"))
   actual <- sort(unique(temp_aloads_recent[temp_aloads_recent$SITE_ABB%in%expected,"SITE_ABB"]))
+  cbind(expected,actual)
```
Comment (cschroed-usgs) [inline tests/testthat/test_annual_load.R]: Does this call to `cbind` influence the test results, or is it for debugging purposes? If the latter, could it be removed?

File: tests/testthat/test_annual_load.R [hunk 3]
```diff
@@ -10,8 +10,13 @@ temp_aloads$FWC_N<-as.numeric(temp_aloads$FWC)
...
 temp_aloads$mod1<-as.character(temp_aloads$MODTYPE)
 temp_aloads[temp_aloads$mod1%in%"REGHIST","mod1"]<-"REG"
-
-
+temp_aloads<-temp_aloads[!is.na(temp_aloads$TONS_L95_N),]
+temp_aloads<-temp_aloads[!is.na(temp_aloads$TONS_U95_N),]
+temp_aloads[temp_aloads$TONS_L95_N>temp_aloads$TONS_U95_N,]
+temp_aloads[temp_aloads$TONS_L95_N>temp_aloads$TONS_N,]
+temp_aloads[temp_aloads$TONS_N>temp_aloads$TONS_U95_N,]
+str(temp_aloads)
```
Comment (cschroed-usgs) [inline tests/testthat/test_annual_load.R]: Does this call to `str` influence the test results, or is it for debugging purposes? If the latter, could it be removed?

File: tests/testthat/test_may_load.R (hunk 1 above)
Comment (supercasey) [reply tests/testthat/test_may_load.R]: Nope, not necessary, this was a mistake (the data release now includes "MONTH", I forgot that we didn't include in the website).

File: tests/testthat/test_annual_load.R (hunk 2 above)
Comment (supercasey) [reply tests/testthat/test_annual_load.R]: Yep, was a relic from testing.

File: tests/testthat/test_annual_load.R (hunk 3 above)
Comment (supercasey) [reply tests/testthat/test_annual_load.R]: Yep, was a relic from testing.

**Comment by supercasey** [comment]
Yes, I made the above changes, will remove commented lines and will resubmit.

**Pull request merged**